opentelemetry-distro/src/opentelemetry/distro/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+
+from opentelemetry.instrumentation.distro import BaseDistro
+
+
+class OpenTelemetryDistro(BaseDistro):
+    """
+    The OpenTelemetry provided Distro configures a default set of
+    configuration out of the box.
+    """
+
+    def _configure(self, **kwargs):
+        os.environ.setdefault("OTEL_EXPORTER", "otlp")
```

Comment:
Is it possible for this to be set AFTER a `Configuration()` call has been made? Should we just directly set it using `Configuration().EXPORTER`?